CHANGELOG.md

```diff
@@ -104,17 +102,19 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Type of parameter `replyMarkup` in method `SendInvoiceAsync` changed to `InlineKeyboardMarkup`
 - Type of parameter `prices` in method `SendInvoiceAsync` changed to `IEnumerable<LabeledPrice>`
 - Type of parameter `shippingOptions` in method `AnswerShippingQueryAsync` changed to `IEnumerable<ShippingOption>`
-- Type of parameter `chatId` in method `SendInvoiceAsync` changed to `long`
+- Type of parameter `chatId` in method `SendInvoiceAsync` changed to `int`
```

Comment:
Why, actually? As far as I understand, chatId must be long, not int.

Reply:
Invoices are always sent in private chats. And private chat ids are always in range of 32 bits.